spec/integration/compare_sirsi_and_folio_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'spec_helper'
 
 describe 'comparing against a well-known location full of documents generated by solrmarc' do
```

Comment:
this describe text doesn't seem right for the test

Reply:
I agree, but I don't think it's appropriate to change the text in this PR?

Reply:
fair!